Chat_App/Chat_App/UI/TypingComponentView.swift

```diff
@@ -0,0 +1,188 @@
+//
+//  TypinComponentView.swift
+//  Chat_App
+//
+//  Created by Gi Oo on 26.04.23.
+//
+
+import UIKit
+
+class TypingComponentView: UIView {
+
+    // MARK: - Properties
+    private var heightConstraint: NSLayoutConstraint? = nil
```

Comment:
```suggestion
    private lazy var heightConstraint = containerView.heightAnchor.constraint(equalToConstant: containerView.frame.height)
```

and just change `isActive` property below